Asset/src/V1/Gapic/AssetServiceGapicClient.php

```diff
@@ -278,6 +427,40 @@ public static function organizationFeedName($organization, $feed)
         ]);
     }
 
+    /**
+     * Formats a string containing the fully-qualified path to represent a
+     * organization_saved_query resource.
+     *
+     * @param string $organization
+     * @param string $savedQuery
+     *
+     * @return string The formatted organization_saved_query resource.
+     */
+    public static function organizationSavedQueryName(
+        $organization,
+        $savedQuery
+    ) {
+        return self::getOrganizationSavedQueryNameTemplate()->render([
+            'organization' => $organization,
+            'saved_query' => $savedQuery,
+        ]);
+    }
+
+    /**
+     * Formats a string containing the fully-qualified path to represent a project
+     * resource.
+     *
+     * @param string $project
+     *
+     * @return string The formatted project resource.
+     */
+    public static function projectName($project)
```

Comment:
This is already defined [here](https://github.com/googleapis/google-cloud-php/pull/5281/files#diff-dd960618bbaf1b500bd6ede5dbc7b9b9ea743b417e18a1de9c1b4d3af883cd22R523). Need to investigate!